### PR TITLE
Willys/back-to-basics-batch-size

### PR DESCRIPTION
**Pull request by wpschaub**
Back to basics post, looking at batch sizes in teh context of building rockets.

### CODE REVIEW

**Review by wpschaub — COMMENTED** [review]
Does the rocket analogy work?

**Review by danielrbroderick — DISMISSED** [review]

File: content/back-to-basics-batch-size.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+- Quick feedback loops.
+- Reduced development times.
+- Simplicity and flexibility.
+- Optimised capacity management.
+
+Let us visualize a few example and spot these advantages.
+
+>
+> ![Build a rocket](/images/back-to-basics-batch-size-alert.png)
+> **CAUTION** - I am intentionally skipping a very important and related topic, which will be covered in the follow-up blog post - **Work in-Progress** (WIP). Without it, leaning towards smaller batch sizes would lead to a nuclear tye reaction (a process in which two particles collide, to produce one or more particles), followed by a similar meltdown.
```
Comment (MerlinDarkstar) [inline content/back-to-basics-batch-size.md]: nuclear tye s/b nuclear type

**Review by MerlinDarkstar — DISMISSED** [review]

**Review by danielrbroderick — DISMISSED** [review]

**Review by AndreKaminski — COMMENTED** [review]
I am not sure if I follow the main point, maybe because the rocket example is a bit hard for me to follow. With the following statement "By reducing the batch size to one, we see magic starting to happen. Previous pros turn into cons, such as quick feedback loops", are you advocating for larger batches? The issue with the example is, as I see it, it contains two major areas - the batch sizing, and reduction of dependencies allowing to work in parallel. However, as you rightly say, there is also serialization here, because the assembly needs to happen in a predefined order. Gene Kim in "Wiring Winning Organizations" describes this as incrementalization, modularization and linearization.

**Comment by wpschaub** [comment]
@AndreKaminski the main point is that instead of a large batch size of work (building the entire rocket), I advocate a batch size of one (building rocket components) - similar to (reusable) components of a solution. Ahhhhh - I see where the confusion comes in - it should read cons turn into pros. Fixed.
https://github.com/wsbctechnicalblog/wsbctechnicalblog.github.io/pull/523

**Comment by wpschaub** [comment]
@AndreKaminski after flipping the pros/cons, does it make more sense now? I need to get this post polished first, as the other two (WIP and Kanban) build on top of this one.

**Review by AndreKaminski — APPROVED** [review]

**Pull request merged**